prometheus/exporter/gpu_exporter.py

```diff
@@ -64,6 +64,13 @@ def collect_gpu_info():
         else:
             logger.exception("command '%s' return with error (code %d): %s",
                     e.cmd, e.returncode, e.output)
+   except OSError as e:
+        if e.errno == os.errno.ENOENT:
+            logger.warning("nvidia-smi not found")
+        else:
+            raise
```

Comment:
Please do not raise here.

If command not exists, this means the user does not want gpu being used by pai, so exporter and health checker will *pretend* gpu not exist in this node. So we just return default value which is None.

Same applies to `healthy_check.py`

Reply:
gotcha and fixed with new commit